lib/app/upload.ex

```diff
@@ -48,8 +44,13 @@ defmodule App.Upload do
       # Fetch the contents of the returned XML string from `ex_aws`.
       # This XML is parsed with `sweet_xml`:
       # github.com/kbrw/sweet_xml#the-x-sigil
+      #
+      # Fetching the URL of the returned file.
       url = upload_response_body.body |> xpath(~x"//text()") |> List.to_string()
-      compressed_url = "#{@compressed_baseurl}#{file_name}"
+
+      # Creating the compressed URL to return as well
+      compressed_bucket_baseurl = "https://s3.eu-west-3.amazonaws.com/#{Application.get_env(:ex_aws, :compressed_bucket)}/"
```

Comment:
The disadvantage of defining these in the function scope is that they must be fetched from the environment on _each_ invocation. is there no way to do this with [`@module_attributes`](https://elixir-lang.org/getting-started/module-attributes.html)?
(_if not, then just leave it like this for now..._)

Reply:
We can do this with `module_attributes` but it will prompt the warning you've mentioned prior. If that's preferable, I can revert this change.

Reply:
I think it's preferable form a latency perspective ... ⌛ 
But it should only add a couple of `ms` to each request so not human-perceptible. 💭